tests/ensure_paths_are_short.py

```diff
@@ -13,6 +13,10 @@ def main():
     ssg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
     max_path = ""
     for dir_, _, files in os.walk(ssg_root):
+        # Don't check for path len of log files
+        # They are not shipped nor used during build
+        if "tests/logs/" in dir_:
```

Comment:
I wonder if something like `linux_os/guide/tests/logs/.../` will also be skipped.

Reply:
It will be skipped.
```
>>> path = 'linux_os/guide/tests/logs/.../'
>>> "tests/logs/" in path
True
```

Reply:
maybe something like this:
```suggestion
        current_relative_path = os.path.relpath(dir_, ssg_root)
        if current_relative_path[:11] == "tests/logs/":
            continue
```

Reply:
@ggbecker Thanks for your suggestion,
Implemented slightly different, but should work the same way.

Reply:
Great. It works perfectly.